deployment/terraform/config.tf

```diff
@@ -1,5 +1,8 @@
 provider "aws" {
   region = var.aws_region
+  default_tags {
```

Comment:
Tags doesn't seem related this anonymisation task. Needs to be pulled out to separate PR